infrastructure/src/main/java/org/corfudb/infrastructure/management/CompleteGraphAdvisor.java

```diff
@@ -94,18 +94,16 @@ public Optional<NodeRank> healedServer(ClusterState clusterState, String localEn
 
         ImmutableList<String> unresponsiveNodes = clusterState.getUnresponsiveNodes();
         if (unresponsiveNodes.isEmpty()) {
-            log.trace("All nodes responsive. Nothing to heal");
             return Optional.empty();
         }
 
         if (!unresponsiveNodes.contains(localEndpoint)) {
             log.trace("Local node is responsive. Nothing to heal");
             return Optional.empty();
         }
-
         //Transform a ClusterState to the ClusterGraph and make it symmetric (symmetric failures)
         ClusterGraph symmetricGraph = ClusterGraph.toClusterGraph(clusterState).toSymmetric();
-
+        log.debug("Symmetric graph: {}", symmetricGraph);
```

Comment:
could you please make it trace?
debug will overflow logs

Reply:
I will leave it for now for debugging purposes. Once its stable in prod, we can remove.